src/bitmessagecli.py

```diff
@@ -756,37 +788,37 @@ def sendMsg(toAddress, fromAddress, subject, message): #With no arguments sent,
                     usrPrompt = 1
                     main()
 
-                for addNum in range (0, numAddresses): #processes all of the addresses
+                for addNum in range(0, numAddresses):  # processes all of the addresses
                     label = jsonAddresses['addresses'][addNum]['label']
                     address = jsonAddresses['addresses'][addNum]['address']
                     #stream = jsonAddresses['addresses'][addNum]['stream']
                     #enabled = jsonAddresses['addresses'][addNum]['enabled']
-                    if (fromAddress == label): #address entered was a label and is found
+                    if (fromAddress == label):  # address entered was a label and is found
                         fromAddress = address
                         found = True
                         break
-                
+
                 if (found == False):
-                    if(validAddress(fromAddress)== False):
+                    if(validAddress(fromAddress) == False):
```

Comment:
Missing space after `if` operator

Reply:
Will be fixed with superfluous-parens issues